data/sql/updates/pending_db_world/rev_1618600341183458700.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1618600341183458700');
+
+DELETE FROM `spell_dbc` WHERE `ID`=20253;
+DELETE FROM `spell_dbc` WHERE `ID`=58620;
+DELETE FROM `spell_dbc` WHERE `ID`=58621;
+DELETE FROM `spell_dbc` WHERE `ID`=64382;
+DELETE FROM `spell_dbc` WHERE `ID`=20252;
+DELETE FROM `spell_dbc` WHERE `ID`=20616;
+DELETE FROM `spell_dbc` WHERE `ID`=20617;
+DELETE FROM `spell_dbc` WHERE `ID`=25272;
+DELETE FROM `spell_dbc` WHERE `ID`=25275;
```

Comment:
DELETE FROM `spell_dbc` WHERE `ID` IN (X, Y, Z);

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/5298/commits/82d18fb5f16b9f494a0b1b5fcb369e83dc7908b4

Even so, the checks continue to fail.